src/Geometry/PUMLReader.cpp

```diff
@@ -59,207 +59,191 @@
 #include <fstream>
 #include <string_view>
 
-class GlobalFaceSorter
-{
-private:
-	const PUML::TETPUML &m_puml;
-
-public:
-	GlobalFaceSorter(const PUML::TETPUML &puml)
-		: m_puml(puml)
-	{ }
-
-	bool operator()(unsigned int a, unsigned int b) const
-	{
-		return m_puml.faces()[a].gid() < m_puml.faces()[b].gid();
-	}
-};
-
 /**
  * @todo Cleanup this code
  */
-seissol::PUMLReader::PUMLReader(const char *meshFile, const char* partitioningLib, double maximumAllowedTimeStep,
+seissol::geometry::PUMLReader::PUMLReader(const char *meshFile, const char* partitioningLib, double maximumAllowedTimeStep,
                                 const char* checkPointFile, initializers::time_stepping::LtsWeights* ltsWeights,
                                 double tpwgt, bool readPartitionFromFile)
-	: MeshReader(MPI::mpi.rank())
+    : seissol::geometry::MeshReader(MPI::mpi.rank())
 {
-	PUML::TETPUML puml;
-	puml.setComm(MPI::mpi.comm());
+    PUML::TETPUML puml;
+    puml.setComm(MPI::mpi.comm());
 
-	read(puml, meshFile);
+    read(puml, meshFile);
   
         generatePUML(puml); // We need to call generatePUML in order to create the dual graph of the mesh
 	if (ltsWeights != nullptr) {
 		ltsWeights->computeWeights(puml, maximumAllowedTimeStep);
 	}
 	partition(puml, ltsWeights, tpwgt, meshFile, partitioningLib, readPartitionFromFile, checkPointFile);
 
-	generatePUML(puml);
+    generatePUML(puml);
 
-	getMesh(puml);
+    getMesh(puml);
 }
 
-void seissol::PUMLReader::read(PUML::TETPUML &puml, const char* meshFile)
+void seissol::geometry::PUMLReader::read(PUML::TETPUML &puml, const char* meshFile)
 {
-	SCOREP_USER_REGION("PUMLReader_read", SCOREP_USER_REGION_TYPE_FUNCTION);
+    SCOREP_USER_REGION("PUMLReader_read", SCOREP_USER_REGION_TYPE_FUNCTION);
 
-	std::string file(meshFile);
+    std::string file(meshFile);
 
-	puml.open((file + ":/connect").c_str(), (file + ":/geometry").c_str());
-	puml.addData((file + ":/group").c_str(), PUML::CELL);
-	puml.addData((file + ":/boundary").c_str(), PUML::CELL);
+    puml.open((file + ":/connect").c_str(), (file + ":/geometry").c_str());
+    puml.addData((file + ":/group").c_str(), PUML::CELL);
+    puml.addData((file + ":/boundary").c_str(), PUML::CELL);
         const auto numTotalCells = puml.numTotalCells();
```

Comment:
- a lot of formating changes, but not consistent with the unchanged lines, it seems.
At this point I would consider applying clang-format.